src/actions/tracker/render_tracker.cr

```diff
@@ -1,15 +1,18 @@
-class TrackerJsAction < BrowserAction
+class RenderTracker < BrowserAction
   include Auth::AllowGuests
 
   get "/js/track.js" do
+    if KindEnv.env("APP_HOST").nil?
+      return head 414
```

Comment:
URI Too Long? :thinking:

Reply:
@alexanderadam - hum, yea. I read 414 as URI Too Short, maybe reading what I want it to be? :D

`412` or `425` might be better. What do you think?

Reply:
I don't know. Maybe `412`

Reply:
I will go with `412` sounded better than 425 as the description of 425 doesn't match as good as 412 does.